### ISSUE TITLE
Unnecessary form validation error message in extension welcome page.

### ISSUE THREAD

**Issue by ivin-3-69** [getAlby/lightning-browser-extension#2260]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

On the Alby extension page - 

> chrome-extension://iokeahhehimjnekafflcihljlcjccdbe/welcome.html

 clicking anywhere gives a form validation error message.
I have attached a video showing the error.

### Screenshots [optional]

Video :
https://user-images.githubusercontent.com/66731461/227512643-0364017a-2c76-42f2-8535-c11ecc488437.mp4

Screenshot of the error message:
https://user-images.githubusercontent.com/66731461/227513236-80c142f4-36c6-484b-b74d-d55cde9e84cf.png

### Steps To Reproduce

1. Visit chrome-extension://iokeahhehimjnekafflcihljlcjccdbe/welcome.html
2. Click anywhere on the screen 
3. You will see the unnecessary error message.

### Expected behavior

When the form is submitted with an empty password field, then only the error message should pop up.

### Alby information

Alby is installed through the chrome extension store.

### Device information

-OS: Ubuntu
-Browser: Chrome
-Version 110.0.5481.177 (Official Build) (64-bit)

### Are you working on this?

None

**Comment by escapedcat** [comment]
The fields are being validated `onBlur` ("leave the input") since [this change](https://github.com/getAlby/lightning-browser-extension/pull/1895/files#diff-b5502e24e705885cd11d000e06333b1a7a9e28459e916d8d0589d18ecc24f466R110).  
@rolznz you remeber the reasoning behind this?

**Comment by ivin-3-69** [comment]
hey, I'm a summer of bitcoin 2023 student. Can I work on this issue? I don't know the SoB formalities. It would be lovely if you could guide me @escapedcat .

**Comment by escapedcat** [comment]
Let's wait for feedback from Roland first

**Comment by ivin-3-69** [comment]
sure

**Comment by rolznz** [comment]
@escapedcat @ivin-3-69 I do not believe this check is unnecessary although could be improved. See below:

Without this check there will be no error message at all if the user tries to click next without filling out the form, since the "Next" button is disabled. This is especially confusing if the user fills out both fields, but accidentally enters a different password in one of them. The next button is disabled and there is no error message showing why.

It would be much better UX if the Next button is always enabled, but if you click it, then it shows an error message if validation fails. Then we don't need to check onBlur and get the weird behaviour mentioned in this issue.

This would happen by default if we used a proper form validation library rather than custom validation code. I created an issue here: https://github.com/getAlby/lightning-browser-extension/issues/1891

What do you guys think? I think fixing this in a way that can be used in the other places around the app would be great, rather than just putting in a custom fix here.

**Comment by rolznz** [comment]
> For me it would be irritating to have the button enabled if the fields are empty.

I think the most important case is when users cannot understand why they cannot submit the form. That's why I think there should always be a helpful error message. It seems obvious if one field is empty that you should fill it, but sometimes people miss it (especially in larger forms with many fields) and the error message can be a second prompt.
